### ISSUE TITLE
Review

### ISSUE THREAD

**Issue by tcrasset** [andypendent/DL-wine-quality#1]
# Structured data review

All the necessary points are there! Good job

## Good

- I like the fact that you separated the EDA into a different file
- Nice use of `stratified` in `train_test_split`
- Nice preprocessing (oversampling, one hot) --> could have been in a separate file too
- The training is successful, the loss is decreasing

## To improve

- Not sure that the formula to compute the number of neurons is really that correct
- Create functions outside of a `.ipynb` file so that you reduce code duplication across different notebooks, e.g. `training`, `compute_score` methods
- What is the conclusion ? What steps improved the score ?
- What is the baseline ?
